tools/bionano/bionano_scaffold_edited.xml

```diff
@@ -0,0 +1,614 @@
+            </when>
+            <when value="">
+            </when>
+        </conditional>
+        <conditional name="quimeric_quality_options">
+            <param name="quimeric_quality" type="select" label="Generate quimeric quality score" help="Flag to generate chimeric quality score for the Input BioNano CMAP">
+                <option value="-y">True</option>
+                <option value="" selected="True">False</option>
+            </param>
+            <when value="-y">
+                <param name="bionano_molecules" type="data" format="bnx" label="Bionano molecules" help="Input BioNano molecules BNX [optional; only required for either the -x or -y option]"/>
+                <param name="noise_parameter" type="data" format="txt" optional="True" label="Noise parameter file" help="Input de novo assembly noise parameter .errbin or .err file [optional; recommended for -y option but not required]"/>
+            </when>
+            <when value="">
+            </when>
+        </conditional>
+        -->
+    </inputs>
+    <outputs>
+        <data name="ngs_contigs_scaffold_fasta" format="fasta" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.fasta" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (fasta)"/>
+        <data name="ngs_contigs_scaffold_agp" format="txt" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.agp" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (agp)"/>
+        <data name="ngs_contigs_scaffold_gap" format="txt" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.gap" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (gap)"/>
+        <data name="report" format="txt" from_work_dir="hybrid_scaffolds/hybrid_scaffold_informatics_report.txt" label="${tool.name} on ${on_string}: hybrid scaffold report"/>
+        <data name="results" format="zip" from_work_dir="results.zip" label="${tool.name} on ${on_string}: results">
+            <filter>zip_file</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="5">
+            <param name="ngs_fasta" value="assembly.fasta.gz"/>
+            <param name="bionano_cmap" value="colormap_assembly.cmap"/>
+            <param name="conflict_filter_genome" value="3"/>
+            <param name="conflict_filter_sequence" value="3"/>
+            <conditional name="configuration_options">
+                <param name="configuration" value="file"/>
+                <param name="configuration_file" value="configuration.xml"/>
+            </conditional>
+            <param name="zip_file" value="true"/>
+            <output name="ngs_contigs_scaffold_fasta" ftype="fasta">
+                <assert_contents>
+                    <has_size value="4753369" delta="300" />
```

Comment:
you can test a FASTA file in a smarter way than size. E.g you can add line count, assert some header line etc in addition.